.env.example

```diff
@@ -13,4 +13,10 @@ MONGODB_CONNECTION_STRING
 
 # SQS
 AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
\ No newline at end of file
+AWS_SECRET_ACCESS_KEY=
+
+# S3
+AWS_S3_KEY_ID=
+AWS_S3_KEY_SECRET=
```

Comment:
so, I think, as long as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are in the env variables, aws sdk automatically fetches the config. like, we never initialise the SQS config in the orchestrator, it just works. although we do use a wrapper library as well so maybe that handles it automatically but i mostly think its the aws sdk itself. and in that case, i think we can just have one aws access key and secret for all AWS services